index.html

```diff
@@ -25,8 +25,8 @@
   </head>
   <body>
     <div id="background">
-        <div class="background-image"></div>
         <div class="background-cover"></div>
+        <div class="background-image"></div>
```

Comment:
Could we keep the order as-is? And then set the `prototype.editor.backgroundOpacity` to a value like `0.8`? The concern I have is that this color might not work well with all color schemes (although it looks great with the default one). If we flip the order back, and use opacity, it will still be present but pretty subtle, and will blend with other colorschemes as well